MapboxNavigation/NavigationComponent.swift

```diff
@@ -37,7 +37,11 @@ public protocol CarPlayConnectionObserver: class {
  */
 public protocol NavigationStatusPresenter: class {
     /**
-     Shows the status view for a specified amount of time.
+     Shows a Status for a specified amount of time.
```

Comment:
I think such comment would be accurate for `func addNewStatus(status: StatusView.Status)`. I think it'd better to comment with `- parameter` statements in this case. Please also add docs for `addNewStatus(status:)` and `hideStatus(status:)`.